season-config.html

```diff
@@ -47,7 +47,7 @@ <h3>Enter in any scoring opportunities in the first text box
 
         <fieldset id = "scoringOpportunities">
```

Comment:
Just a thought, how would we track notes or other free text fields?, probably keep this as is, but put on the "future improvements list" to allow for selecting the input type.